lib/widget/sfWidgetFormSchema.class.php

```diff
@@ -822,7 +822,7 @@ public function moveField($field, $action, $pivot = null)
         array_unshift($this->positions, $field);
         break;
       case sfWidgetFormSchema::LAST:
-        array_push($this->positions, $field);
+        $this->positions []= $field;
```

Comment:
I used to write it like this: `$this->positions[] = $field`. But this syntax definitely makes sense.